terraform/router/local/main.tf

```diff
@@ -43,6 +21,21 @@ resource "kubernetes_service" "router" {
       target_port = 443
     }
 
-    selector = module.nginx.selector
```

Comment:
why is this one not needed anymore?
and could we just remove the selector altogether or we need the "convox-dummy" one?

Reply:
removed the selector